### ISSUE TITLE
harbor input doesn't work with Cool Mic android app - Feeding stopped: Not_found

### ISSUE THREAD

**Issue by makeworld-the-better-one** [savonet/liquidsoap#1154]
**Describe the bug**
I am not able to stream audio from my phone using the Android app. Streaming from another application like butt works fine however.

Here is the relevant part of the log, when I try to stream from Cool Mic:
```
2020/04/12 20:00:30 [decoder:3] Method "OGG" accepted "application/ogg".
2020/04/12 20:00:30 [input(dot)harbor_9148:3] Decoding...
2020/04/12 20:00:30 [input(dot)harbor_9148:2] Feeding stopped: Not_found.
```
When I stream from butt, which works:
```
2020/04/12 20:02:32 [decoder:3] Method "OGG" accepted "audio/ogg".
2020/04/12 20:02:32 [input(dot)harbor_9148:3] Decoding...
2020/04/12 20:02:33 [fallback_9161:3] Switch to amplify_9158 with transition.
2020/04/12 20:02:41 [input(dot)harbor_9148:2] Feeding stopped: Ogg.End_of_stream.
```
The main difference seems to be the MIME type. Does liquidsoap not know the MIME type `application/ogg`?

Here are the relevant lines of my script:
```liq
set("harbor.icy_formats",["audio/mpeg","audio/aacp","audio/aac","audio/x-aac","audio/wav","audio/wave","audio/x-flac", "audio/ogg", "audio/vorbis", "application/ogg"])
mic = audio_to_stereo(nrj(input.harbor("mic", port = 8006, password = "passwordhere", buffer = 1.0)))
```
**Expected behavior**
The audio comes in, there are no errors in the log, and streaming from Cool Mic works the same as butt.

**Version details**
 - OS: Debian stable (Buster)
 - Version 1.4.1

**Install method**
I installed liquidsoap from Debian testing. [Here's a link.](https://packages.debian.org/bullseye/liquidsoap)

**Comment by makeworld-the-better-one** [comment]
It seems the default setting of Opus in Cool Mic is the problem. I guess my liquidsoap build doesn't work with Opus? Anyway switching to Ogg Vorbis output in Cool Mic fixed the problem, and I was able to stream. Here all the log lines:
```
2020/04/12 22:41:01 [decoder:3] Method "OGG" accepted "application/ogg".
2020/04/12 22:41:01 [input(dot)harbor_9148:3] Decoding...
2020/04/12 22:41:03 [fallback_9161:3] Switch to amplify_9158 with transition.
2020/04/12 22:41:12 [input(dot)harbor_9148:2] Feeding stopped: Ogg.End_of_stream.
```

**Comment by makeworld-the-better-one** [comment]
However, I tried Ogg Vorbis with Cool Mic before, and it didn't work. Turns out the real problem was I was sending a mono stream from Cool Mic, but in my script the input was wrapped with `audio_to_stereo(nrj(`, and apparently `nrj` doesn't work with mono streams?? Switching to stereo streaming in Cool Mic, or changing the function order to `nrj(audio_to_stereo(`, both fixed my problem. I went with the latter.
